### PR TITLE
cleaned Select from hardcoded Strings

### PR DESCRIPTION
**Pull request by RoyMontoya**
Using hard-coded Strings is not recommended, constant Strings are easier to understand and the code looks cleaner.

Also changed some of the if conditionals to a cleaner look.

### CODE REVIEW

**Comment by RoyMontoya** [comment]
we should change the name of method getWhereCond to getWhereClause, since is a getter for the variable whereClause

File: library/src/test/java/com/orm/query/SelectTest.java [hunk 1]
```diff
@@ -35,6 +35,34 @@ public void testWhere(){
         assertEquals("2", where.getArgs()[1]);
     }
 
+    @Test
+    public void toSqlAllClauses(){
+        String toSql = Select.from(TestRecord.class)
+                .where("test")
```
Comment (rilopez) [inline library/src/test/java/com/orm/query/SelectTest.java]: could you add more meaningful names  instead of `"test"`?

**Comment by RoyMontoya** [comment]
@sibeliusseraphini is there any chance that someone could check this and other pullrequest pending?

**Comment by sibeliusseraphini** [comment]
@RoyMontoya I will try to take a look this week

**Comment by RoyMontoya** [comment]
@sibeliusseraphini  ok! thanks!

**Comment by sibeliusseraphini** [comment]
@RoyMontoya merged thanks, sorry for the delay

**Pull request merged**